www/js/domoticz.js

```diff
@@ -6317,13 +6317,14 @@ function WatchLiveSearch(){
 				var name=$(this).find('#name').html().toUpperCase();
 				var desc=$(this).find('#name').attr('data-desc');
 				var idx=$(this).find('#name').attr('data-idx');
+				var status=$(this).find('#name').attr('data-status').toUpperCase();
```

Comment:
Just checking, if (for example for a utility devices) there is no status, would this not give an assertion?
Maybe status should be checked like 'desc' below, if it is undefined, we make it empty else we make it uppercase?

Reply:
my bad! You're right!

_--> I always forget that there is currently no global template for devices 😉_ 

A fix will follow ASAP.

Reply:
fixed!